tensorrt_llm/_torch/pyexecutor/sampler.py

```diff
@@ -578,15 +578,15 @@ def setup_sampler_step(self, requests):
                 decoder_requests)
 
     @staticmethod
-    def beam_width(scheduled_requests: ScheduledRequests) -> int:
```

Comment:
I think it makes sense to type this as `Iterable[LlmRequest]` if you're going to pass the chain of `.all_requests`, and rename to requests

Reply:
Done.